plugins/sqlfluff-plugin-example/src/example/rules.py

```diff
@@ -68,15 +68,16 @@ class Rule_Example_L001(BaseRule):
         ORDER BY bar
     """
 
-    config_keywords = ["forbidden_columns"]
```

Comment:
Nitpick: Why was this moved below the `__init__` function?

Reply:
No good reason -- I'll move it up